pandas/tests/indexes/datetimes/test_constructors.py

```diff
@@ -543,29 +542,23 @@ def test_construction_outofbounds(self):
         ]
         exp = Index(dates, dtype=object)
         # coerces to object
-        tm.assert_index_equal(Index(dates), exp)
+        tm.assert_index_equal(Index(dates).astype(object), exp)
```

Comment:
Why do we need object here?

Reply:
good catch, makes more sense to change `exp`.  will update